pyrenew/convolve.py

```diff
@@ -217,3 +217,45 @@ def compute_delay_ascertained_incidence(
         mode="valid",
     )
     return delay_obs_incidence
+
+
+def daily_to_epiweekly(
+    daily_value: ArrayLike,
```

Comment:
Suggest naming this argument `daily_values` plural since it will typically be a timeseries.